docs/contributing.md

```diff
@@ -3,3 +3,82 @@
 ## Translating
 
 Translation is managed through [Transifex](https://www.transifex.com/yohanboniface/umap/dashboard/).
+
+## Bugs Triaging
+
+You are very welcome to help us triaging [uMap issues](https://github.com/umap-project/umap/issues).
+
+* Help other users by answering questions
+* Give your point of view in discussions
+* And so on...
+
+## Development on Ubuntu
+
+### Environnement setup
+
+Follow the procedure [Ubuntu from scratch](ubuntu.md)
+
+But instead using folders /etc/umap, you can create a ~/.umap folder.
+This folder will contain the umap.conf file.
+
+And for folder /srv/umap, you can create a ~/umap folder (We will remove this folder later)
+
+You will have to set an env var, we will set it in your .bashrc:
+
+    nano ~/.bashrc
+
+Add the following at the end of file:
+
+```
+# uMap
+export UMAP_SETTINGS=~/.umap/umap.conf
```

Comment:
For local hacking, you can use a `local.py` file inside the `settings` directory, it will be used as a fallback

Reply:
For local hacking, see https://umap-project.readthedocs.io/en/latest/install/

Reply:
Oops, merged before approval ;)

I think this part is a duplicate of "install" and thus should be removed and a link made to there.
What do you think ?

Reply:
Ooooops :shipit:
You're right, will change it next time :pig: